scripts/include-sf.js

```diff
@@ -4,26 +4,32 @@ const path = require('path');
 const shelljs = require('shelljs');
 const fs = require('fs');
 
-const sfUnixPath = 'node_modules/@salesforce/cli/bin/run';
+const npmGlobalBin = shelljs.exec('npm list -g --depth 0 | head -1').stdout.trim();
+const sfGlobalPath = path.join(npmGlobalBin, 'node_modules', '@salesforce', 'cli');
+
+// Copy globally installed sf to sf-cli/ at root of sfdx
+shelljs.mv('-f', sfGlobalPath, path.join('..', 'sf-cli'));
```

Comment:
So close I can almost taste it.... but I don't know what's going on here.
I added the `path.join('..', 'sf-cli')` in an attempt to make sure this doesn't get caught in the `tmp` folder defined by cli:vtarballs:verify in plugin-release-management [here](https://github.com/salesforcecli/plugin-release-management/blob/main/src/commands/cli/tarballs/verify.ts#L54).

Maybe I just need to tweak this a little more. I can't figure out how that `tmp` folder disappears though and what happens to the files in it when the tarballs are actually built because I don't see it in the ones you can download from [the install page](https://developer.salesforce.com/docs/atlas.en-us.234.0.sfdx_setup.meta/sfdx_setup/sfdx_setup_install_cli.htm).

Reply:
@RasPhilCo would you be able to enlighten me about how the `tmp` directory is used when oclif builds a tarball for a cli and what happens to it and the files in it before the tarball is "finalized"?

If not I can do some more spelunking tomorrow.